docs/company/charter.md

```diff
@@ -0,0 +1,76 @@
+# Charter
+
+Thoth Tech charter is important for steering our team in the right direction,
```

Comment:
@tancnle and if we can just adjust this to "The Thoth Tech charter" also?

Reply:
Raised a follow-up issue https://github.com/thoth-tech/handbook/issues/6